user-interface/src/main/java/life/qbic/datamanager/views/projects/create/AddProjectDialog.java

```diff
@@ -210,6 +213,11 @@ private boolean isDialogContentValid() {
   }
 
   private boolean isDialogContentInvalid() {
+    //Since we don't employ one Binder for all fields within a layout anymore,
```

Comment:
Why not work with binder validation here? You could have a simple record with three contacts?

Reply:
This method employs the binder validation on a `ContactField` specific level, since the binding is now done within each `BoundContactField` in the `CollaboratorLayout` and not the `CollaboratorLayout` itself, which is why this workaround is necessary.